pkg/shell/hosts.jsx

```diff
@@ -46,6 +56,24 @@ function HostLine({ host, user }) {
     );
 }
 
+// top left navigation element when host switching is disabled
+export const CockpitCurrentHost = ({ machine }) => {
+    const [user, setUser] = useState("");
+
+    useInit(() => {
+        cockpit.user()
```

Comment:
There is also `useLoggedInUser` in hooks.ts. Could that be used?

Reply:
Oh yes, absolutely. Thanks for pointing out.

Reply:
Ah, forgot about that. Updated to useLoggedInUser